tests/inspect/inspect.go

```diff
@@ -112,13 +114,21 @@ func main() {
 			}
 			c := capability.Cap(capInt)
 			if caps.Get(capability.BOUNDING, c) {
-				fmt.Printf("%v=enabled", c.String())
+				fmt.Printf("%v=enabled\n", c.String())
 			} else {
-				fmt.Printf("%v=disabled", c.String())
+				fmt.Printf("%v=disabled\n", c.String())
 			}
 		}
 	}
 
+	if globalFlags.PrintUser {
+		// gexpect is racy: workaround it here
+		// TODO: fix it in gexpect, see: https://github.com/ThomasRooney/gexpect/pull/5
+		time.Sleep(10 * time.Millisecond)
```

Comment:
The fix https://github.com/ThomasRooney/gexpect/pull/6 is merged.
I will update this...